### PR TITLE
KAA-1256: Separate Tenant and User management

### PR DESCRIPTION
**Pull request by Pyshankov**

### CODE REVIEW

**Comment by kaa-leeroy** [comment]
Can one of the admins verify this patch?

**Comment by Oldnickus** [comment]
@kaa-leeroy add to whitelist

File: pom.xml [hunk 1]
```diff
@@ -75,6 +75,8 @@ Copyright 2014-2016 CyberVision, Inc.
         <commons-logging.version>1.1.1</commons-logging.version>
         <httpcomponents.version>4.3.2</httpcomponents.version>
         <jackson.version>2.4.1</jackson.version>
+        <javax.validation.version>1.1.0.Final</javax.validation.version>
+        <hibernate.validator.version>5.2.2.Final</hibernate.validator.version>
```
Comment (ashvayka) [inline pom.xml]: Where is this used?

File: server/node/src/main/java/org/kaaproject/kaa/server/control/service/DefaultControlService.java [hunk 2]
```diff
@@ -363,52 +363,15 @@ public void deleteUser(String userId) throws ControlServiceException {
...
-     * (non-Javadoc)
-     * 
-     * @see
-     * org.kaaproject.kaa.server.control.service.ControlService#editTenantAdmin
-     * (org.kaaproject.kaa.common.dto.TenantAdminDto)
-     */
     @Override
-    public TenantAdminDto editTenantAdmin(TenantAdminDto tenantAdmin) throws ControlServiceException {
-        return userService.saveTenantAdmin(tenantAdmin);
+    public List<UserDto> findAllTenantAdminsByTenantId(String tenantId) throws ControlServiceException {
```
Comment (ashvayka) [inline server/node/src/main/java/org/kaaproject/kaa/server/control/service/DefaultControlService.java:369]: Please make sure that you have corresponding REST API call and method in AdminClient

**Pull request closed without merging**